graylog2-web-interface/src/pages/LUTDataAdaptersPage.jsx

```diff
@@ -95,48 +96,58 @@ const LUTDataAdaptersPage = createReactClass({
   },
 
   render() {
+    const { route: { action } } = this.props;
+    const {
+      dataAdapter,
+      validationErrors,
+      types,
+      dataAdapters,
+      pagination,
+      tableStore,
+    } = this.state;
     let content;
-    const isShowing = this.props.route.action === 'show';
-    const isEditing = this.props.route.action === 'edit';
+    const isShowing = action === 'show';
+    const isEditing = action === 'edit';
 
     if (isShowing || isEditing) {
-      if (!this.state.dataAdapter) {
+      if (!dataAdapter) {
         content = <Spinner text="Loading data adapter" />;
       } else if (isEditing) {
         content = (
           <Row className="content">
             <Col lg={12}>
-              <h2>Data Adapter</h2>
-              <DataAdapterForm dataAdapter={this.state.dataAdapter}
-                               type={this.state.dataAdapter.config.type}
+              {/* <h2>Data Adapter</h2> */}
```

Comment:
I think this should be removed from here, since we are rendering the title in the `DataAdapterForm` now.

Reply:
Fixed